packages/terra-site/webpack.config.js

```diff
@@ -6,6 +6,7 @@ const path = require('path');
 const autoprefixer = require('autoprefixer');
 const ExtractTextPlugin = require('extract-text-webpack-plugin');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
+const I18nAggregatorPlugin = require('./node_modules/terra-i18n/lib/I18nAggregatorPlugin');
```

Comment:
In here, it's a little hard to separate the plugin and I18n. In general, I18n component is a dependency and Plugin should be a devDependency. I don't have some ideas in here, but we may want to have some discussions on it.